pkg/api/api.go

```diff
@@ -276,15 +277,15 @@ func requestPipelineFn(s storage.Storer, r *http.Request) pipelineFunc {
 // calculateNumberOfChunks calculates the number of chunks in an arbitrary
 // content length.
 func calculateNumberOfChunks(contentLength int64, isEncrypted bool) int64 {
-	if contentLength < 4096 {
+	if contentLength < swarm.ChunkSize {
```

Comment:
should  this  not  be <=?

Reply:
This whole function was just copied from `swarm` project (with constants replaced).
Will change this.